### ISSUE TITLE
Angular 14+ error:  ERROR TypeError: Cannot read properties of undefined (reading 'nativeElement')

### ISSUE THREAD

**Issue by Sampath-Lokuge** [scaleflex/ng-cloudimage-responsive#33]
Hi,

Cloud image is not working with Angular 14+.

**Error**

```
core.mjs:7644 ERROR TypeError: Cannot read properties of undefined (reading 'nativeElement')
    at ImgComponent.processImage (ng-cloudimage-responsive.mjs:326:38)
    at ng-cloudimage-responsive.mjs:226:22
    at timer (zone.js:3155:47)
    at _ZoneDelegate.invokeTask (zone.js:443:35)
    at Object.onInvokeTask (core.mjs:26367:33)
    at _ZoneDelegate.invokeTask (zone.js:442:64)
    at Zone.runTask (zone.js:214:51)
    at ZoneTask.invokeTask (zone.js:525:38)
    at ZoneTask.invoke (zone.js:514:52)
    at data.args.<computed> (zone.js:3135:36)
```

**package.json**

```
"dependencies": {
    "@angular/common": "^14.0.0",
    "@angular/core": "^14.0.0",
    "@angular/forms": "^14.0.0",
    "@angular/platform-browser": "^14.0.0",
    "@angular/platform-browser-dynamic": "^14.0.0",
    "@angular/router": "^14.0.0",
    "@capacitor/app": "4.0.1",
    "@capacitor/core": "4.0.1",
    "@capacitor/haptics": "4.0.1",
    "@capacitor/keyboard": "4.0.1",
    "@capacitor/status-bar": "4.0.1",
    "@ionic/angular": "^6.1.9",
    "ng-cloudimage-responsive": "^3.0.1",
    "rxjs": "~6.6.0",
    "tslib": "^2.2.0",
    "zone.js": "~0.11.4"
  },
```

**Git Repo:** 

https://github.com/Sampath-Lokuge/cloud-image-issue

**Files:** 

https://github.com/Sampath-Lokuge/cloud-image-issue/blob/main/src/app/home/home.page.html

https://github.com/Sampath-Lokuge/cloud-image-issue/blob/main/src/app/app.module.ts

https://github.com/Sampath-Lokuge/cloud-image-issue/blob/main/src/app/home/home.module.ts

**Steps to reproduce:**

> Clone the repo

> npm i

> ionic serve

**Comment by amrw-js** [comment]
hey, @Sampath-Lokuge  
fixed in [v4.0.0](https://github.com/scaleflex/ng-cloudimage-responsive/releases/tag/v4.0.0)

**Comment by Sampath-Lokuge** [comment]
Hi @amrw-js 

Yes, it is working fine now. Thanks a lot!
